### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by BradReesWork**
* added Hungarian to api.rst
* fixed missing examples
* fixed syntax

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun tests

File: python/cugraph/dask/centrality/katz_centrality.py
Comment (rlratzel) [inline python/cugraph/dask/centrality/katz_centrality.py:119]: minor: since these are typically copy-and-pasteable snippets, you could use a comment char here:
```
>>> import cugraph.dask as dcg
>>> # ... Init a DASK Cluster
>>> #   see https://docs.rapids.ai/api/cugraph/stable/dask-cugraph.html
>>> chunksize = dcg.get_chunksize(input_data_path)
```

**Review by rlratzel — APPROVED** [review]
LGTM. I had just one minor suggestion which need not hold up my approval.

**Comment by BradReesWork** [comment]
rerun tests

File: python/cugraph/dask/centrality/katz_centrality.py
Comment (BradReesWork) [reply python/cugraph/dask/centrality/katz_centrality.py:119]: true.  but it is not executable code.  There used to be a line initializing Comm, but that is not fully what is needed for DASK.  I pulled that out so that there is one place to maintain what setting up dask looks like.  If someone cut&pastes the above block, I would prefer it break on a non-executable line

**Review by afender — APPROVED** [review]
:+1:

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**
